pisi/cli/__init__.py

```diff
@@ -37,7 +37,7 @@ def printu(obj, err = False):
         out = sys.stderr
     else:
         out = sys.stdout
-    out.write(obj.encode('utf-8'))
+    out.write(str(obj))
```

Comment:
The conversion isn't required since `if not isinstance(obj, str)` already ensures `obj` is a string.